server/controllers/groups/create.js

```diff
@@ -13,13 +13,17 @@ const sanitization = {
   searchable: {
     optional: true,
     generic: 'boolean'
+  },
+  open: {
+    optional: true,
+    generic: 'boolean'
```

Comment:
```suggestion
    generic: 'boolean'
    default: false
```
my mistake, boolean parameter have no default default value, so it need to be set: the tests fail otherwise